themes/bootstrap3/templates/admin/menu.phtml

```diff
@@ -5,7 +5,5 @@
   <a href="<?=$this->url('admin/maintenance')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'maintenance' ? ' active' : ''?>"><?=$this->transEsc('System Maintenance')?></a>
   <a href="<?=$this->url('admin/tags')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'tags' ? ' active' : ''?>"><?=$this->transEsc('Tag Maintenance')?></a>
   <a href="<?=$this->url('admin/feedback')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'feedback' ? ' active' : ''?>"><?=$this->transEsc('Feedback Management')?></a>
-  <?php if ($this->overdrive()->showOverdriveAdminLink()):?>
-    <a href="<?=$this->url('admin/overdrive')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'overdrive' ? ' active' : ''?>"><?=$this->transEsc('Overdrive API')?></a>
-  <?php endif;?>
+  <a href="<?=$this->url('admin/overdrive')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'overdrive' ? ' active' : ''?>"><?=$this->transEsc('od_admin_menu')?></a>
```

Comment:
Is there a reason why we made the Overdrive menu item unconditional here? Might we still want the ability to disable it when it's not relevant?

Reply:
Well the permission wasn't working for some reason and I noticed none of the others had something like that so I changed it to be like them. If Overdrive is not configured, it just displays empty values which seems appropriate to me. But let me know if you want me to put it back and make it work again.

Reply:
This is probably why it wasn't working: [VUFIND-1398](https://openlibraryfoundation.atlassian.net/browse/VUFIND-1398)

Reply:
I took a couple of minutes to look into this and pushed up commit 3a3bd264fe51d6b65e069a808b433e53e7837a89 to reinstate the configuration. I think it's worth having, since people may have varying levels of comfort with providing web-based access to data about their settings, etc. Of course, feel free to adjust if you don't like anything I've done here.